src/style.css

```diff
@@ -1 +1,144 @@
 /* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Although this is an exception, it's better to avoid setting global styles such as `margin`, `padding`, and `box-sizing` on the `body` tag. It's more common to set these on the `*` selector. But, as per the checklist, using `*` selector is not recommended due to its impact on performance. So, it's better to set these properties on individual elements as needed.